### DISCUSSION TITLE
Configure dynamically lua_max_pending_timers

### DISCUSSION THREAD

**Discussion by secat** [Kong/kong#7319] [Ideas and feature requests]
Is there a way currently with Kong v2.0.4 to configure dynamically `lua_max_pending_timers` in the `nginx_kong.lua` file?

**Comment by hbagdi** [comment]
The only possible way is to use a custom nginx template: https://docs.konghq.com/2.0.x/configuration/.

We should make this configurable using nginx directive injection mechanism: https://docs.konghq.com/2.0.x/configuration/#injecting-nginx-directives
